Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ however, it has to be formatted properly to pass verification tests.
 
 ### Changed
 - Improved performance of HID descriptor parsing by moving json parsing to a simple custom predicitve parser instead of relying on Unity's json parsing. This should improve domain reload times when there are many HID devices connected to a machine.
+- Fixed `ArgumentNullException` when opening the Prefab Overrides window and selecting a component with an `InputAction`.
```

Comment:
Shouldn't this should be under the [Unreleased] section?

Reply:
I'm in the middle of the release just now, so develop doesn't have the [Unreleased] section right now. That's probably why it's went here.
I will move this into [Unreleased] after I put it back and bump to v1.4.4 for the next release.